packages/@coorpacademy-components/src/organism/brand-form/index.js

```diff
@@ -42,7 +44,18 @@ function BrandForm(props, context) {
 
   const disabledSubmit = disabled || isPending || !isModified;
   const submitButton = onSubmit ? (
-    <div className={style.saveButton}>
+    <div
+      data-tip={tooltip && tooltip.title ? tooltip.title : ''}
```

Comment:
You don't need to check the tooltip object here. Use defaultProps instead.

Reply:
done